tests/bluetooth/bsim_bt/compile.sh

```diff
@@ -61,3 +61,4 @@ app=tests/bluetooth/bsim_bt/bsim_test_audio compile
 app=tests/bluetooth/bsim_bt/edtt_ble_test_app/hci_test_app compile
 app=tests/bluetooth/bsim_bt/edtt_ble_test_app/gatt_test_app compile
 app=tests/bluetooth/bsim_bt/bsim_test_mesh compile
+app=tests/bluetooth/bsim_bt/bsim_test_mesh_pst compile
```

Comment:
Could it be done as low latency tests with a separate configuration instead of a separate folder?
https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bluetooth/bsim_bt/compile.sh#L66

Reply:
I guess this increases the folder hierarchy depth. I think this would work if @trond-snekvik is fine with it.

Reply:
+1 for this suggestion. You can reference CONFIG_XXX in cmakelists.txt and C to make the persistent storage modules only be compiled in if `CONFIG_BT_SETTINGS` is enabled.